sdk/identity/identity/test/playwright/rollup/src/index.ts

```diff
@@ -0,0 +1,4 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export { InteractiveBrowserCredential } from "../../../../src/credentials/interactiveBrowserCredential.browser";
```

Comment:
If I were to do this in another project, other than our SDK repo, I would load here the `@azure/identity` package, like:
```ts
export { InteractiveBrowserCredential } from "@azure/identity";
```